doc/release/1.3.2-notes.rst

```diff
@@ -10,8 +10,58 @@ support for Python 3.8.
 Authors
 =======
 
+* CJ Carey
+* Martin Gauch +
+* Ralf Gommers
+* Matt Haberland
+* Huize +
```

Comment:
@huizew You may like for us to update the mailmap file if you'd like your name displayed differently in our release notes?

Reply:
@tylerjereddy Thanks. I would like to use full name `Huize Wang`.

Reply:
Ok,  updated the mailmap & regenerated the relnotes accordingly.